.travis.yml

```diff
@@ -26,7 +26,9 @@ before_install:
   - cd WcaOnRails/
 before_script:
   - bundle exec rake db:reset
-  - bundle exec rake assets:precompile
+  # See https://github.com/usabilityhub/rails-erb-loader/issues/63#issuecomment-374791504
+  # and our https://github.com/thewca/worldcubeassociation.org/issues/3476
+  - DISABLE_SPRING=1 bundle exec rake assets:precompile
```

Comment:
@jfly I couldn't quite figure out why it wouldn't work within the preloader call, but it seems to work from here.
Context information: executing `rake` doesn't start spring, but executing `rails runner` (when compiling webpacker packs) does.
Spring's doc indicate [how to temporarily not use Spring](https://github.com/rails/spring#temporarily-disabling-spring), but I wasn't sure if it would be passed down to the child process actually running the `rails runner`.

What I did to check that:
```bash
bin/spring stop
rm -rf public/packs
bundle exec rake assets:precompile
# Open htop meanwhile and notice spring was started
bin/spring stop
rm -rf public/packs
DISABLE_SPRING=1 bundle exec rake assets:precompile
# Notice that spring doesn't start in htop
```

So as far as I can tell it *does* disable spring for assets compilation, so I'm tempted to go ahead with this.
Sounds good?